### PR TITLE
gh #217 L1 dsVideoDevice Issue fix

### PR DESCRIPTION
**Pull request by akhilbhas**
* Addressign issues reported in 3.1.2 release

### CODE REVIEW

**Review by hari22yuva — DISMISSED** [review]
ship it

**Review by KarthikeyanR470 — COMMENTED** [review]
@akhilbhas Could you please check once whether below API's are addressed as mentioned in JIRA MTK-98?. 

1. test_l1_dsVideoDevice_positive_dsGetDFC()
2. test_l1_dsVideoDevice_positive_dsGetCurrentDisplayframerate

**Comment by akhilbhas** [comment]
> 2\. test_l1_dsVideoDevice_positive_dsGetCurrentDisplayframerate

HI @KarthikeyanR470 ,
These were already addressed in https://github.com/rdkcentral/rdk-halif-test-device_settings/pull/142

**Review by srinivasgtl — APPROVED** [review]
ship it

File: src/test_l1_dsVideoDevice.c [hunk 1]
```diff
@@ -1004,11 +1021,11 @@ void test_l1_dsVideoDevice_positive_dsForceDisableHDRSupport(void)
 
         // Step 03: Force disable HDR support (set to true)
         result = dsForceDisableHDRSupport(handle, true);
-        UT_ASSERT_EQUAL(result, dsERR_OPERATION_NOT_SUPPORTED);
+        UT_ASSERT_EQUAL(result, dsERR_NONE);
```
Comment (hari22yuva) [inline src/test_l1_dsVideoDevice.c:1024]: as per spec changes is not valid
https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsVideoDevice.h#L283

File: src/test_l1_dsVideoDevice.c [hunk 2]
```diff
@@ -1004,11 +1021,11 @@ void test_l1_dsVideoDevice_positive_dsForceDisableHDRSupport(void)
...
 
         // Step 04: Force disable HDR support (set to false)
         result = dsForceDisableHDRSupport(handle, false);
-        UT_ASSERT_EQUAL(result, dsERR_OPERATION_NOT_SUPPORTED);
+        UT_ASSERT_EQUAL(result, dsERR_NONE);
```
Comment (hari22yuva) [inline src/test_l1_dsVideoDevice.c:1028]: as per spec changes is not valid 
https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsVideoDevice.h#L283

**Review by hari22yuva — COMMENTED** [review]
check these comments

File: src/test_l1_dsVideoDevice.c (hunk 2 above)
Comment (akhilbhas) [reply src/test_l1_dsVideoDevice.c]: updated

File: src/test_l1_dsVideoDevice.c (hunk 1 above)
Comment (akhilbhas) [reply src/test_l1_dsVideoDevice.c]: updated

**Review by hari22yuva — APPROVED** [review]

**Review by KarthikeyanR470 — APPROVED** [review]
Looks good

**Pull request merged**